src/Microsoft.AspNetCore.Mvc.ViewFeatures/exceptions.net45.json

```diff
@@ -1,10 +1,21 @@
 [
     {
+        "OldTypeId": "public interface Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper",
+        "OldMemberId": "Microsoft.AspNetCore.Html.IHtmlContent TextBox(System.String current, System.Object value, System.String format, System.Object htmlAttributes)",
+        "Kind": "Removal"
+    },
+    {
+        "OldTypeId": "public interface Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper",
```

Comment:
FYI I filed aspnet/BuildTools#217 about the need for this property in an addition.